pkg/chain/ethereum/config.go

```diff
@@ -45,6 +45,14 @@ type Config struct {
 	// allowed gas price is reached, no further resubmission attempts are
 	// performed.
 	MaxGasPrice uint64
+
+	// RequestsPerSecondLimit sets the maximum average number of requests
+	// per second which can be executed against the Ethereum node.
+	RequestsPerSecondLimit int
+
+	// ConcurrencyLimit sets the maximum number of concurrent requests which
+	// can be executed against the Ethereum node in the same time.
```

Comment:
For the same reason as mentioned in the previous comment, I'd improve the documentation a little:
```
// This limit affects all types of chain requests, including view function calls.
```